lib/splitclient-rb/validators.rb

```diff
@@ -1,8 +1,11 @@
 # frozen_string_literal: true
+require 'set'
 
 module SplitIoClient
   class Validators
 
+    Flagset_regex = /^[a-zA-Z0-9][-_.:a-zA-Z0-9]{0,79}$/
```

Comment:
where did this regex come from? 😱 
it should be `"^[a-z0-9][_a-z0-9]{0,49}$"`

Reply:
Hmmm.. this is used in Python SDK, I will correct both.